portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,28 +32,45 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> messages;
+  private List<String> comments; //update to custom struct later
```

Comment:
Is this field still needed? It looks like you're no longer adding elements to it in `init()` and you're fetching comments using DatastoreService now.